### PR TITLE
Fix unused --test-batch-size command line argument

### LINKED ISSUE #157: two lines of code in mnist/main.py
There are two arguments called batch_size and test_batch_size:
`parser.add_argument('--batch-size', type=int, default=64, metavar='N',
                    help='input batch size for training (default: 64)')`
`parser.add_argument('--test-batch-size', type=int, default=1000, metavar='N',
                    help='input batch size for testing (default: 1000)')`
but batch_size is used here:
`test_loader = torch.utils.data.DataLoader(
    datasets.MNIST('../data', train=False, transform=transforms.Compose([
                       transforms.ToTensor(),
                       transforms.Normalize((0.1307,), (0.3081,))
                   ])),
    batch_size=args.batch_size, shuffle=True, **kwargs)`

Also, what does this line(line 105) do:
`test_loss = test_loss`

and it seems that `epoch` is not used in test().

### PR DESCRIPTION
**Pull request by pbelevich**
Fixes #157

### CODE REVIEW

**Comment by soumith** [comment]
thanks!
